ArduCopter/RC_Channel_Copter.cpp

```diff
@@ -633,7 +636,9 @@ bool RC_Channel_Copter::do_aux_function(const AuxFuncTrigger &trigger)
 
 #if AP_COPTER_AHRS_AUTO_TRIM_ENABLED
         case AUX_FUNC::AHRS_AUTO_TRIM:
+            if (copter.flightmode->allows_auto_trim()) {
             copter.g2.rc_channels.do_aux_function_ahrs_auto_trim(ch_flag);
+            }
             break;
```

Comment:
```suggestion
            if (!copter.flightmode->allows_auto_trim()) {
                break;
            }
            copter.g2.rc_channels.do_aux_function_ahrs_auto_trim(ch_flag);
            break;
```

... means the indenting isn't broken

Reply:
done